projects/visual-regression/src/app/components/input/input.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="page">
+  <h3 tsVerticalSpacing>
+    Input with Datepicker
+  </h3>
+
+  <ts-input
+    [(ngModel)]="myDatepickerValue"
+    label="Input with datepicker"
+    mask="date"
+    name="Datepicker input"
+    tsVerticalSpacing="small--0"
+    [datepicker]="true"
+  ></ts-input>
+
+
+  <h3 tsVerticalSpacing>
+    Input: Reactive Form
+  </h3>
+
+  <button (click)="shouldDisable = !shouldDisable">
+    {{ shouldDisable ? 'Enable' : 'Disable' }} the name input
+  </button>
+
+  <form [formGroup]="myForm" novalidate>
+    <br>
+    <ts-input
+      [label]="label"
+      [formControl]="myForm.get('readonly')"
+      [readOnly]="true"
+    ></ts-input>
+    <ts-input
+      [formControl]="myForm.get('name')"
+      [label]="label1"
+      [isClearable]="clearable"
+      hint="Name input with validation"
+      [isDisabled]="shouldDisable"
+      [spellcheck]="false"
+      [prefixIcon]="icon"
+      #foo="tsInput"
+    ></ts-input>
+
+    <ts-input
+      [formControl]="myForm.get('email')"
+      [label]="label2"
+      [isClearable]="clearable"
+      [prefixIcon]="'alternate_email'"
+      [validateOnChange]="true"
+      [hint]="'Email input with validation'"
+      name="email"
+      [spellcheck]="false"
+      [autocomplete]="'email'"
+      type="email"
+    ></ts-input>
+
+    <ts-input
+      [formControl]="myForm.get('phone')"
+      label="Phone"
+      hint="Phone input with masking"
+      mask="phone"
+    ></ts-input>
+  </form>
+
+
+  <h3 tsVerticalSpacing>
+    Input: Textarea
+  </h3>
+
+  <ts-input
+    [(ngModel)]="textareaModel"
+    label="My textarea"
+    name="My textarea"
+    hint="So much space! How will you use it?"
+    [isTextarea]="true"
+  ></ts-input>
+
+  <strong>ngModel value:</strong>
```

Comment:
Should our visual tests be checking model values?

Reply:
Do you mean the actual model value or what's displayed on UI? It's already checking UI value..

Reply:
Meaning, I'm not sure why `visual regression` tests are checking model values. That seems like something our standard tests should cover leaving only visual items for these tests.

Reply:
Ah, wasn't trying to test the actual model values. Was just thinking presentation in UI might be compared (updated versus benchmark). Not an important piece and certainly can move out.

Reply:
yeah, imo it's extra to look at and doesn't add any value